src/tests/UnitTests/EnsureEnumParamTests.cs

```diff
@@ -9,29 +9,121 @@ public class EnsureEnumParamTests : UnitTestBase
         [Fact]
         public void IsDefined_ShouldNotThrow()
         {
-            var item = Only1IsValid.Valid;
+            var item = EnumDummy.Valid;
 
             ShouldNotThrow(
                 () => Ensure.Enum.IsDefined(item, ParamName),
                 () => EnsureArg.EnumIsDefined(item, ParamName),
                 () => Ensure.That(item, ParamName).IsDefined());
         }
 
+        [Theory]
+        [InlineData((EnumDummy)2)]
+        [InlineData((EnumDummy)0)]
```

Comment:
Not to be defensive of the original code, but `Only1IsValid` makes the call sites (like NotDefined_ShouldThrow) a lot more easy to read at the call site to know that 0 and 2 are not defined ;)

Reply:
Okay, will revert to your name. 😉